src/lib/viewers/BaseViewer.js

```diff
@@ -587,6 +587,9 @@ class BaseViewer extends EventEmitter {
 
         if (this.annotator && this.areNewAnnotationsEnabled()) {
             this.annotator.emit(ANNOTATOR_EVENT.setVisibility, true);
+            this.annotator.toggleAnnotationMode(
```

Comment:
Should this instead be
```
if (this.options.enableAnnotationsDiscoverability) {
    this.annotator.toggleAnnotationMode(AnnotationMode.REGION);
}
```
?

Reply:
Yeah that makes sense - we probably don't need to call with NONE again if the FF is off since it's already NONE.